response_operations_ui/views/messages.py

```diff
@@ -271,7 +289,7 @@ def _refine(message):
 
 
 def _get_survey_id(selected_survey):
-    return survey_controllers.get_survey_id_by_short_name(selected_survey)
+    return [survey_controllers.get_survey_id_by_short_name(selected_survey)]
```

Comment:
Why has this changed in this PR?  Wasn't this part of Adams work with the different FDI survey ID's?

Reply:
I don't know why I changed this? It should work with or without being a list, but I'll change back